go.mod

```diff
@@ -564,6 +564,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/lorenzosaino/go-sysctl v0.3.1
```

Comment:
This dependency adds a lot of indirect packages. Can you use our existing `pkg/network/config/sysctl` package instead?

Reply:
no, as the `pkg/network/config/sysctl` does not provide the option to Set values

Reply:
You can run the command `sysctl -w net.ipv4.tcp_timestamps=1` or write to `/proc/sys/net/ipv4/tcp_timestamps` to accomplish this (we already do this for a different sysctl in a test). Adding a dependency for this simple operation seems overkill IMHO.